src/lib/components/Grid/index.js

```diff
@@ -504,6 +506,10 @@ const GridBase = memo(({
             path += id;
             dispatchData({ type: 'UPDATE_FORM_MODE', payload: '' })
         }
+        if (model?.addUrlParamKey) {
```

Comment:
Why we get from model?.addUrlParamKey if we already destrucered the addUrlParamKey

Reply:
Why ? u should handle by passing default value
Example 
```
const {record={}}=data;
record[addUrlParamKey] || ''
```